src/storage/quota/GlobalQuotaStrategy.ts

```diff
@@ -0,0 +1,70 @@
+// These two eslint lines are needed to store 'this' in a variable so it can be used
+// in the PassThrough of trackAvailableSpace
+/* eslint-disable @typescript-eslint/no-this-alias */
+/* eslint-disable consistent-this */
+import { PassThrough } from 'stream';
+import type { RepresentationMetadata } from '../../http/representation/RepresentationMetadata';
+import type { ResourceIdentifier } from '../../http/representation/ResourceIdentifier';
+import { PayloadHttpError } from '../../util/errors/PayloadHttpError';
+import type { Guarded } from '../../util/GuardedStream';
+import { guardStream } from '../../util/GuardedStream';
+import type { Size } from '../size-reporter/Size';
+import type { SizeReporter } from '../size-reporter/SizeReporter';
+import type { QuotaStrategy } from './QuotaStrategy';
+
+/**
+ * The GlobalQuotaStrategy sets a limit on the amount of data stored on the server globally
+ */
+export class GlobalQuotaStrategy implements QuotaStrategy {
```

Comment:
We could perhaps also create a structure like `config/storage/quota/global.json`, `config/storage/quota/pods.json` ... ?

Reply:
Could also be an option, but atm I would lean towards not doing this since quota only works with the file system so it might cause confusion why it doesn't work in the other cases.